client/src/styles/home.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
Combing the styles into one stylesheet called style.css

Reply:
Combined!